conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -321,9 +321,9 @@ func (algodImp *algodImporter) GetBlock(rnd uint64) (data.BlockData, error) {
 				delta, err = algodImp.getDelta(rnd)
 				if err != nil {
 					if status.LastRound < rnd {
```

Comment:
why not do this check right after getting the status?  i.e. what algodImp.mode is this not valid for

Reply:
Now I'm wondering if this is even possible, since `StatusAfterBlock` should ensure the Conduit round is not ahead of algod.